src/hooks/useUniversalRouter.ts

```diff
@@ -0,0 +1,80 @@
+import { BigNumber } from '@ethersproject/bignumber'
+// eslint-disable-next-line @typescript-eslint/no-restricted-imports
```

Comment:
This should not be disabled. Instead, it can import the type or import through the abstract package.